### PR TITLE
feat(tools): add visability of auth0 rules

### PR DESCRIPTION
**Pull request by Bouncey**
In looking into the `accountLinkId` issue described in #107, I found it was as simple as adding `uuid()` inside the rule

### CODE REVIEW

**Comment by raisedadead** [comment]
Quick query are you using the webtask integration (natively provided by auth0) for this? We can move this to "Auth0 Management API + Lambda" in future for more complex actions.

File: auth0-rules/add-accountLinkId-to-token.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  Function name should be removed for deployment
+
+  uuid is global inside Auth0 Rules
+
+  https://auth0.com/docs/appliance/modules
+
+*/
+
+function addAccountLinkId(user, context, callback) {
+  const namespace = 'https://www.freecodecamp.org/accountLinkId';
```
Comment (raisedadead) [inline auth0-rules/add-accountLinkId-to-token.js]: This is just a URL and need not have any actual endpoint?

Comment (Bouncey) [reply auth0-rules/add-accountLinkId-to-token.js]: Yes, purely a namespace to avoid collisions. No requests are made to it.

**Comment by Bouncey** [comment]
Whilst trying to access the `app_metadat` from the token, I stumbled across [this comment](https://community.auth0.com/t/the-app-metadata-is-not-included-in-id-token/7830/2) which lead me to this rule

File: auth0-rules/add-accountLinkId-to-token.js (hunk 1 above)
Comment (raisedadead) [reply auth0-rules/add-accountLinkId-to-token.js]: In that case I would prefer making this to something like `https://auth-ns.freeCodeCamp.org`

File: src/auth/index.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ const { JWT_CERT } = process.env;
 
 export { updateAppMetaData } from './auth0';
 
-export const namespace = 'https://www.freecodecamp.org/';
+export const namespace = 'https://auth-ns.freecodecamp.org/';
```
Comment (ojongerius) [inline src/auth/index.js:11]: Just curious and don't have too much context of Auth0, but why `auth-ns`?

Comment (raisedadead) [reply src/auth/index.js:11]: Its just a namespace that has to be any URL. For the purposes of keeping us sane, its short for auth namespace as `auth-ns`. There will never be an actual endpoint or anything here.

Comment (raisedadead) [reply src/auth/index.js:11]: This is kind of useful for huge orgs with lots of separate sub entities, like SSO, Windows AD, etc. where there will be a need for namespaces, but we dont't care about that for our purposes.

**Review by raisedadead — APPROVED** [review]

File: src/auth/index.js (hunk 2 above)
Comment (ojongerius) [reply src/auth/index.js:11]: Ah right. Could have been freecodecamp.org but this give us some flexibility in regards to future changes in authentication. Thanks @raisedadead !

**Pull request merged**